tgui/packages/tgui/interfaces/MarionetteRemote.js

```diff
@@ -0,0 +1,163 @@
+                        disabled={selected_command !== "say" && selected_command !== "emote"}
+                        content={entered_data ? entered_data : "UNSET"}
+                      />)
+                      : (
+                        <>
+                          <Button
+                            onClick={() => act("set_data", { new_data: "NORTH" })}
+                            icon="arrow-up"
+                            selected={entered_data === "NORTH"}
+                          />
+                          <Button
+                            onClick={() => act("set_data", { new_data: "SOUTH" })}
+                            icon="arrow-down"
+                            selected={entered_data === "SOUTH"}
+                          /><Button
+                            onClick={() => act("set_data", { new_data: "WEST" })}
+                            icon="arrow-left"
+                            selected={entered_data === "WEST"}
+                          /><Button
+                            onClick={() => act("set_data", { new_data: "EAST" })}
+                            icon="arrow-right"
+                            selected={entered_data === "EAST"}
+                          />
+                        </>)}
+                  </LabeledList.Item>
+                  <LabeledList.Item key="command" label="Command">
+                    <Button
+                      onClick={() => act("set_command", { new_command: "say" })}
+                      content="Say"
+                      selected={selected_command === "say"}
+                    />
+                    <Button
+                      onClick={() => act("set_command", { new_command: "emote" })}
+                      content="Emote"
+                      selected={selected_command === "emote"}
+                    />
+                    <Button
+                      onClick={() => act("set_command", { new_command: "step" })}
+                      content="Step"
+                      selected={selected_command === "step"}
+                    />
+                    <Button
+                      onClick={() => act("set_command", { new_command: "drop" })}
+                      content="Drop"
+                      selected={selected_command === "drop"}
+                    />
+                    <Button
+                      onClick={() => act("set_command", { new_command: "use" })}
+                      content="Use"
+                      selected={selected_command === "use"}
+                    />
+                    <Button
+                      onClick={() => act("set_command", { new_command: "shock" })}
+                      content="Shock"
+                      selected={selected_command === "shock"}
+                    />
+                  </LabeledList.Item>
+                  <LabeledList.Item key="action_heat" label="Heat Per Action">
+                    {selected_command === "shock" || selected_command === "drop" ? "HIGH"
```

Comment:
Spitballing ideas, maybe this could be done in a tooltip, or maybe red/yellow/green colors and a thunder icon would be nice here.

Reply:
I like it as a separate readout personally, fits the clunky remote feeling